### PR TITLE
Tests: Improve shouldContainExactlyInAnyOrder assertions

### PR DESCRIPTION
**Pull request by fviernau**
Note: If this PR should get accepted a similar change for `shouldContainExactly` should be implemented.

### CODE REVIEW

File: downloader/src/funTest/kotlin/vcs/GitWorkingTreeTest.kt [hunk 1]
```diff
@@ -86,7 +86,9 @@ class GitWorkingTreeTest : StringSpec() {
         }
 
         "Git correctly lists remote branches" {
-            val expectedBranches = listOf(
+            val workingTree = git.getWorkingTree(zipContentDir)
```
Comment (sschuberth) [inline downloader/src/funTest/kotlin/vcs/GitWorkingTreeTest.kt:90]: Commit message: Both ticks should be backticks.

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt [hunk 2]
```diff
@@ -273,27 +274,24 @@ class SpdxExpressionTest : WordSpec() {
...
             "work with LicenseRef-* identifiers" {
                 "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0 AND LicenseRef-scancode-commercial-license"
-                    .decompose() shouldContainExactlyInAnyOrder
-                        listOf(
-                            "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
-                            "LicenseRef-scancode-commercial-license"
-                        )
+                    .decompose() should containExactlyInAnyOrder(
+                    "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
+                    "LicenseRef-scancode-commercial-license"
```
Comment (sschuberth) [inline spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:293]: Wrong indentation.

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt [hunk 3]
```diff
@@ -303,7 +301,7 @@ class SpdxExpressionTest : WordSpec() {
 
             "not merge license-exception pairs with single matching licenses" {
                 "GPL-2.0-or-later WITH Classpath-exception-2.0 AND GPL-2.0-or-later"
-                    .decompose() shouldContainExactlyInAnyOrder listOf(
+                    .decompose() should containExactlyInAnyOrder(
```
Comment (sschuberth) [inline spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:304]: Wrong indentation below. I acknowledge it was broken before, but I believe it makes sense to fix it now.

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt (hunk 2 above)
Comment (fviernau) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:293]: Hm, `as-is` formatting is what I get from the IDE. Do you have any hint?

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt (hunk 3 above)
Comment (fviernau) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:304]: Same as above.

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt (hunk 2 above)
Comment (sschuberth) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:293]: Odd, but I can confirm the IDE to *not* complain about the current formatting. However, IMO it should be

                    .decompose() should containExactlyInAnyOrder(
                        "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
                        "LicenseRef-scancode-commercial-license"
                    )

Comment (fviernau) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:293]: If I do that and apply auto-formatting afterwards the IDE undoes the change. Anyhow, let me fix it up as your proposed formatting looks better.

Comment (fviernau) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt]: Done.

File: spdx-utils/src/test/kotlin/SpdxExpressionTest.kt (hunk 3 above)
Comment (fviernau) [reply spdx-utils/src/test/kotlin/SpdxExpressionTest.kt:304]: Done.

**Pull request merged**